### ISSUE TITLE
Fluid regulator with pumps on the same pipeline causes the pipes to cease updating

### ISSUE THREAD

**Issue by tommy307507** [GTNewHorizons/GT-New-Horizons-Modpack#8037]
#### Which modpack version are you using?
2.1.0.7a
#
#### If in multiplayer; On which server does this happen? 
Single player 
#
#### What did you try to do, and what did you expect to happen?
Pump out of tank, goes to pipe, go thu fluid regulator , go to large steam turbine  , profit

![image](https://user-images.githubusercontent.com/49487505/121725526-f468da80-cb1b-11eb-9eb8-6b5c3df3bbb6.png)

#

#### What happened instead? (Attach screenshots if needed)
The pipes are not updating and needs to be updated or replaced to regular gt fluid mechanics to work
this is a half-full pipe connected to another pipe that has 4000 mb when it ceased to work at the time that I connected 
![image](https://user-images.githubusercontent.com/49487505/121726053-9ab4e000-cb1c-11eb-99ae-656b96c7b1fa.png)
this 2 block pipe section to the rest of the stuff

![image](https://user-images.githubusercontent.com/49487505/121725839-59bccb80-cb1c-11eb-8722-e66bb51dac9a.png)

![image](https://user-images.githubusercontent.com/49487505/121725810-51649080-cb1c-11eb-9f5d-33d343f1a1dc.png)
when I disconnect all other pipes and leave them together the fluid doesnt move at all
![image](https://user-images.githubusercontent.com/49487505/121725924-7527d680-cb1c-11eb-84ef-fbb9490ae46f.png)
breaking all the affected pipes made them work again
#
#### What do you suggest instead/what changes do you propose?
make pumps work with fluid regulators again ?

#
#### What is your GTNH Discord username?
Hebi

**Comment by Prometheus0000** [comment]
Do you mean pipeline or pipe? Are they both on the same pipe block?
Does it work when they are not?
Did this used to work? If so, which version did it work in?

**Comment by tommy307507** [comment]
By pipeline I mean the entire connected section of pipe blocks
they aren't , I tried them being on same block first and didn't work, 
then I reset every block, get the regulator working again and the turbine spinning, 
then tried this configuration and it doesn't work anymore like the same block configuration

tldr: same block or not it didnt work, ALL connected pipes stops updating

I didnt test previous versions of this, to test it I suppose it would be just use 2 tanks and transfer using pump and insert using regulators 🤷
